test/integration/dialects/postgres/dao.test.js

```diff
@@ -465,6 +465,35 @@ if (dialect.match(/^postgres/)) {
           expect(user.length).to.equal(1);
         });
 
+        it('should be able to insert a new record even with an array of enums in a schema', async function() {
+          const User = this.sequelize.define('UserEnums', {
+            schema: 'specialSchema',
```

Comment:
schema is part of the options right? So it needs to be in the second array. And possibly the schema needs to be defined before as well, but not sure about that.

Reply:
Yes, you are right, and the schema is needed, I already have it on my local db, but I will add it on the test.

Reply:
The updated test is now passing. A [Flaky] test for mysql8/node16 failed but this fix is postgres specific, I don't think is realted, I checked the test and is using enums or anything related from what I see.

Reply:
Yeah, the flaky means that it fails sometimes for no real reason. I will rerun the checks for you

Reply:
I saw I had a typo on my last commit, fix(31804) instead of fix(13804), if the merges are squash and merge, is probably fine, if not, let me know and I can ammed the commit and force push to this branch with the new message